<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="QueryOrderTest"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.706">
  <properties />
            <testcase name="orderpaytype" time="0.706" classname="cloud.order.query.OrderPayType">
                <failure type="java.sql.SQLException"
                  message="After end of result set"
              >
<![CDATA[java.sql.SQLException: After end of result set
  com.mysql.jdbc.SQLError.createSQLException(SQLError.java:965)
  com.mysql.jdbc.SQLError.createSQLException(SQLError.java:898)
  com.mysql.jdbc.SQLError.createSQLException(SQLError.java:887)
  com.mysql.jdbc.SQLError.createSQLException(SQLError.java:861)
  com.mysql.jdbc.ResultSetImpl.checkRowPos(ResultSetImpl.java:790)
  com.mysql.jdbc.ResultSetImpl.getStringInternal(ResultSetImpl.java:5213)
  com.mysql.jdbc.ResultSetImpl.getString(ResultSetImpl.java:5136)
  com.mysql.jdbc.ResultSetImpl.getString(ResultSetImpl.java:5175)
  org.zt.common.MysqlJdbc.Postdata(MysqlJdbc.java:62)
  cloud.order.query.OrderPayType.orderpaytype(OrderPayType.java:28)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)
  org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
  org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:579)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
